pkg/phases/base/test.go

```diff
@@ -19,12 +20,12 @@ func Test(platform *platform.Platform, test *console.TestResults) {
 		k8s.TestNamespace(client, "quack", test)
 	}
 
-	if platform.Nginx == nil || !platform.Nginx.Disabled {
+	if (platform.Nginx == nil || !platform.Nginx.Disabled) && (platform.Contour == nil || platform.Contour.Disabled )  {
 		k8s.TestNamespace(client, "ingress-nginx", test)
 	}
 
 	if platform.Minio == nil || !platform.Minio.Disabled {
 		k8s.TestNamespace(client, "minio", test)
 	}
-
+	contour.Test(platform, test)
```

Comment:
this should be tested inline e.g.

```golang
if platform.GetIngressController() == "contour") {
    k8s.TestNamespace(client, "contour")
} else {
    k8s.TestNamespace(client, "nginx-ingress")
}
```
The tests would only be considered a skip if both nginx and contour are disabled.